R/release.R

```diff
@@ -112,8 +112,6 @@ release_checklist <- function(version, on_cran) {
       Check if any deprecation processes should be advanced, as described in \\
       [Gradual deprecation](https://lifecycle.r-lib.org/articles/communicate.html#gradual-deprecation)",
       type != "patch" && has_lifecycle),
-    todo("Bump required R version in DESCRIPTION to {tidy_min_r_version}",
```

Comment:
Now in upkeep, rather than release.

Reply:
I kinda like it as part of the release checklist, but not a hill I will die on.

Reply:
I don't have strong feelings either except that it should be in one or the other.